readded everything (#104)
MrLongbottom/P10-Article: sections/Abstract.tex

> We introduce models that extend the \gls{lda} to include author and category metadata information and a third model which applies taxonomy metadata on the \gls{pam}.

Author: ```suggestion
We introduce topic models that extend the \gls{lda} to include author and category metadata information and a third model which applies taxonomy metadata on the \gls{pam}.
```

Reviewer: ```suggestion
We introduce topic models that extend the \gls{lda} to include author and category metadata information and a third model which applies taxonomy metadata on the \gls{pam}.
```

> The author-topic and category-topic models are based on the author-topic model from \citet{author_topic_2012} with slight modifications, and the taxonomy-topic model uses the \gls{pam} from \citet{li2006pachinko}.

Reviewer: vi må ikke have cites i abstract

> The author-topic and category-topic models are based on the author-topic model from \citet{author_topic_2012} with slight modifications, and the taxonomy-topic model uses the \gls{pam} from \citet{li2006pachinko}.

Author: ```suggestion
The author-topic and category-topic models are based on the author-topic model with slight modifications, and the taxonomy-topic model uses the \gls{pam}.
```

To make the \gls{pam} include the metadata information in each layer of its \gls{dag} structure, a novel topic locking mechanism is created.

> The results show that for the applied dataset, the author-topic and category-topic models get a lower topic coherence than \gls{lda}. \vejleder[inline]{focus on the good results rather than the bad}

Reviewer: ```suggestion
The results show that for the applied dataset, the modified pachinko model, we have made integrates meta-data well and improves the run-time.
```

Author: ```suggestion
The results show that for the applied dataset, our modified pachinko model integrates the meta-data well and improves the run-time.
```

However, the taxonomy-topic model has a higher topic coherence and more understandable topics than \gls{lda}.

> For each model, we also give proposals for how they can be used for recommendation in a news article environment.

Reviewer: ```suggestion
```
